website/blog/2021-05-15-tungsten.md

```diff
@@ -0,0 +1,212 @@
+---
+author: Tomasz Godzik
+title: Metals v0.10.3 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.3, which brings support for
+the long awaited Scala 3.0.0 version together with some important improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">64</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">29</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">6</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">10</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">5</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/38?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Scala 3.0.0 support!
+- Extract class code action.
+- Rename file as class.
+- [Scala 3] Auto imports in completions and code actions.
+- Support same names in different build targets.
+
+## Extract class code action
+
+Due to the great work by [@Giggiux](https://github.com/Giggiux) Metals now
+provides a new `Extract to file` code action, which is available when there are
+multiple definitions in the source file. This works for classes, traits, enums,
+and objects. The code action will not show if the definition we are extracting
+is or extends a `sealed` definition.
+
+![gif1](https://i.imgur.com/dKKkLcL.gif)
+
+The extraction happens via a code action on the definition's name, it will also
+extract any relevant package definition, imports, and, if available, the
+companion.
```

Comment:
I'd maybe recommend just bumping this sentence up to the above one before the gif. It seems like it's just floating alone here.